website/src/app/blog/how-dns-works-in-firezone/readme.mdx

```diff
@@ -385,7 +385,8 @@ works the same for any third-party SaaS service.
 
 1. An admin defines a DNS Resource with address `*.slack.com` in the Firezone
    admin portal. Notice the wildcard -- this will route all subdomains for Slack
-   through Firezone as well, which helps ensure all relevant Slack traffic is routed.
+   through Firezone as well, which helps ensure all relevant Slack traffic is
+   routed.
```

Comment:
Is there a hard word wrap I should set in my editors?

Reply:
So I use prettier set at 80, but it's not a hard requirement. Wrapping markdown is handled to varying degrees of success depending on the editor.

Reply:
I would just like to avoid like 200 line markdown paragraphs if possible.